src/backend/expungeservice/crawler/parsers/case_parser.py

```diff
@@ -0,0 +1,24 @@
+from html.parser import HTMLParser
+
+from expungeservice.crawler.models.charge import Charge
+
+
+class CaseParser(HTMLParser):
+
+    def __init__(self):
+        HTMLParser.__init__(self)
+        self.charges = []
```

Comment:
I think this line isn't used. I was originally going to use it, and decided I agree with what @a-ilango was suggesting and create the charge classes from outside of the parser. I will remove it now.